src/platform/OpenThread/GenericThreadStackManagerImpl_OpenThread.cpp

```diff
@@ -1002,6 +1026,211 @@ CHIP_ERROR GenericThreadStackManagerImpl_OpenThread<ImplClass>::_JoinerStart(voi
     return error;
 }
 
+#if CHIP_DEVICE_CONFIG_ENABLE_THREAD_SRP_CLIENT
+
+static_assert(OPENTHREAD_API_VERSION >= 80, "SRP Client requires a more recent OpenThread version");
+
+template <class ImplClass>
+void GenericThreadStackManagerImpl_OpenThread<ImplClass>::OnSrpClientNotification(otError aError,
+                                                                                  const otSrpClientHostInfo * aHostInfo,
+                                                                                  const otSrpClientService * aServices,
+                                                                                  const otSrpClientService * aRemovedServices,
+                                                                                  void * aContext)
+{
+    switch (aError)
+    {
+    case OT_ERROR_NONE: {
+        ChipLogProgress(DeviceLayer, "OnSrpClientNotification: Last requested operation completed successfully");
```

Comment:
Doesn't it flood the logs with unnecessary information? If so, consider `ChipLogDetail`.

Reply:
Not exactly, as probably it will be displayed only once, per each service. And in most cases we will use only one service for SRP.